sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/ContainerClient.java

```diff
@@ -189,6 +190,15 @@ public URL getContainerUrl() {
         return containerAsyncClient.getContainerUrl();
     }
 
+    /**
+     * Gets the {@link HttpPipeline} powering this client.
+     *
+     * @return The pipeline.
+     */
+    public HttpPipeline getHttpPipeline() {
+        return containerAsyncClient.getHttpPipeline();
+    }
```

Comment:
Do we have an issue to add these to Files and Queues as well?

Reply:
It will be needed for the builder feature additions.